### DISCUSSION TITLE
Can Casdoor run at URL subpath, or only root domain?

### DISCUSSION THREAD

**Discussion by wearpants** [casdoor/casdoor#2941] [Q&A] [answered]
I'd like to run Casdoor at a subpath (like `https://mydomain.com/casdoor`) , using Nginx as a reverse proxy (my use case is on an airgapped network, and I'd like to minimize the number of domain names & SSL certs we need to provision)

I tried various settings including `origin`, `staticBaseUrl` and `ServerUrl` (in frontend's Setting.js) and cannot get this to work (bad links, 404'd files, etc).

Is this possible, and if so how?

**Comment by hsluoyz** [comment, marked as answer]
@wearpants subpath is not supported. You can use subdomain

**Reply by wearpants** [reply]
This should be documented somewhere.
